escaping single quote character as &apos

nramyasri-sf opened the pull request.

nramyasri-sf committed:
escaping single quote

ashpurik commented on dom.js:
```diff
@@ -409,7 +409,8 @@ function _xmlEncoder(c){
          c == '&' && '&amp;' ||
-         c == '"' && '&quot;' ||
+	 c == '"' && '&quot;' ||
+	 c == "'" && '&apos;' ||
```
This should be fine. Just add the indentations so it aligns :) The only thing is I had been worried about adding too many characters to this list in case there were unintended consequences and this being a third party library. But if the tests are passing, and no issues arise, it should be fine. So just try to monitor any issues of this nature going forward to make sure there are no regressions.

nramyasri-sf replied:
Thanks so much @ashpurik . I will keep an eye out for any regressions.

nramyasri-sf committed:
adding indentations

nramyasri-sf committed:
adding indentations

adding indentations

adding indentations

adding indentations

adding indentations

adding indentations
